lib/kvdb/kvdb_meta.c

```diff
@@ -668,10 +668,9 @@ kvdb_meta_from_mpool_cparams(
     for (int i = HSE_MCLASS_BASE; i < HSE_MCLASS_COUNT; i++) {
         const char *mc_path = params->mclass[i].path;
 
-        /* strnlen() + 1 should move us past the final trailing / */
         strlcpy(meta->km_storage[i].path,
```

Comment:
Please make sure the "paths" smoke test passes with this change.. That tests a wide variety of home directory configurations...

Reply:
Btw, @gsramdasi tells me that the "paths" test is broken on master.. It should be one of the recent commits as it was working fine for me mid last week...

Reply:
Cool. It's probably related to the mentioned bug. I'll give it a look